synapse/api/auth.py

```diff
@@ -454,6 +450,7 @@ def _verify_third_party_invite(self, event, auth_events):
                         decode_base64(public_key)
                     )
                     verify_signed_json(signed, server, verify_key)
+
```

Comment:
Might be worth commenting that we don't check that the correct server has signed it. That happens elsewhere.